ons_alpha/release_calendar/models.py

```diff
@@ -143,7 +151,7 @@ def toc(self):
         items = [{"url": "#summary", "text": _("Summary")}]
 
         if self.status == ReleaseStatus.PUBLISHED:
-            for block in self.content:  # pylint: disable=not-an-iterable
+            for block in self.content or []:
```

Comment:
Question: What's the reason for the `or []`? `self.content` should always be iterable.